adamsjr8576/swapi-trivia
# Router app

### What does this PR do?
- This PR Adds fixes some styling issues that arose from implementing Route to the project
- when you wrap an element it a <Link> tag it is actually wrapping it in a <a> tag, so you need to restyle accordingly.
- It creates styling to display the character cards and movie scroll
- implements a back to movies button that links back to :movies and empties the state.characters array.
- Adds Star Wars Logo png and displays in header.

### How should these be tested?
- Fill out the form on page load, check state, and click login. navigate around site and test the logout functionality, back to movies, viewing characters and all should be bug free.

### Issues
- No new issues from this PR. There are still some things that could be styled a little better but we can table those till later. I think it would be really cool to add some animations/gifs to our loading pages and error page which we have discussed already

### Screenshots 

![screencapture-localhost-3000-2019-12-06-13_48_50](https://user-images.githubusercontent.com/49846853/70355516-9628b080-182f-11ea-9988-886c98e2456b.png)


<img width="1440" alt="Screen Shot 2019-12-06 at 1 50 22 PM" src="https://user-images.githubusercontent.com/49846853/70355529-9f198200-182f-11ea-84bb-a26c0961ae6f.png">

@Garrett-Iannuzzi
